Packs/CommonScripts/Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py

```diff
@@ -56,7 +56,7 @@ def get_fqdn(the_input):
     fixed = get_tld(the_input, fail_silently=True, as_object=True, fix_protocol=True)
     domain = fixed or get_tld(the_input, fail_silently=True, as_object=True)
 
-    if domain and domain.tld != 'zip':
+    if domain:
```

Comment:
let's mention in a comment the removal and why.

Reply:
Done